backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/acct/api/AcctSchemaGeneralLedger.java

```diff
@@ -50,6 +52,10 @@ public class AcctSchemaGeneralLedger
 	@NonNull
 	Account purchasePriceVarianceOffsetAcct;
 
+
+	@Nullable
+	Account cashRoundingAcct;
```

Comment:
can't we have this account not null?
Or in case it's complex, I would suggest introducing a method called getCashRoundingAcctNotNull() which throws something like "Account CASH_ROUNDING_ACCT was not set for accounting schema"....